public/js/admin_profile.js

```diff
@@ -65,6 +67,9 @@ ready(async function () {
                 console.log(response.msg);
             } else {
```

Comment:
this function should be added to caller_profile.js and responder_profile.js, otherwise profile photos won't be shown even if uploaded.